buildspec.yml

```diff
@@ -0,0 +1,20 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      java: corretto11
```

Comment:
This should be corretto8 to make sure we can still build/run on Java 8

Reply:
Made the change

Reply:
Hooked up to github-codebuild-logs, getting a SignatureDoesNotMatch error, but apparently according to this https://github.com/jlhood/github-codebuild-logs/issues/15 the issue gets resolved by itself after a few days